apk-generator/v2/app/static/js/main.js

```diff
@@ -245,4 +255,8 @@ $form.submit(function (e) {
         .catch(function () {
             showError();
         });
+    
+    socket.on('message', function(message) {
```

Comment:
First error is a js error. Socket is not defined in this scope. You need to bind it to the socket after it gets connected.